src/api/app/controllers/webui/staging/workflows_controller.rb

```diff
@@ -20,6 +20,8 @@ def create
     staging_workflow = @project.build_staging
     authorize staging_workflow
 
+    staging_workflow.managers = Group.find_by(title: params[:managers_id])
```

Comment:
[nitpick] Why ``managers_id`` and not ``title`` (or ``managers_title`` or ``group_title``

Reply:
Yes, sure I will change it